### PR TITLE
Add KudasaiJP validator to rehearsal-4

### PR DESCRIPTION
**Pull request by liray-unendlich**
## Description

Please include a detailed description of the changes being proposed in this pull request.

**Validators:**
- KudasaiJP
- Add genesis account + gentx

## Type of Change

- [*] New Validator
- [ ] Governance Proposal
- [ ] Bug fix
- [ ] Documentation update
- [ ] Tooling enhancement

## Testing

- [*] I've read [CONTRIBUTING.md](../CONTRIBUTING.md)
- [ ] I ran `make validate-genesis`
- [ ] I ran `make verify-hashes`
- [*] I ran gentx command with our own methods 

## Additional Information

Any additional information that you want to provide.

## Reviewers:

/assign @froch

Thank you for supporting the Burnt Networks!

### CODE REVIEW

File: testnets/xion-rehearsal-4/gentx/KudasaiJP.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+					"max_change_rate": "0.100000000000000000"
+				},
+				"min_self_delegation": "1",
+				"delegator_address": "xion18494yxx84qrt5a8y33uppuzgrfkq77q0jt8nhj",
+				"validator_address": "xionvaloper18494yxx84qrt5a8y33uppuzgrfkq77q0egv4lp",
+				"pubkey": {
+					"@type": "/cosmos.crypto.ed25519.PubKey",
+					"key": "H/eJx3GdqesrqoTa3FtXptMYT8oySi6lWTwes/SxknA="
+				},
+				"value": { "denom": "axion", "amount": "1000000000000" }
```
Comment (froch) [inline testnets/xion-rehearsal-4/gentx/KudasaiJP.json]: this denomination doesn't exist (`axion`)

**Review by froch — CHANGES_REQUESTED** [review]
please regenerate your gentx with `1000000uxion`

File: testnets/xion-rehearsal-4/gentx/KudasaiJP.json (hunk 1 above)
Comment (liray-unendlich) [reply testnets/xion-rehearsal-4/gentx/KudasaiJP.json]: Updated, thanks!

Comment (froch) [reply testnets/xion-rehearsal-4/gentx/KudasaiJP.json]: thank you!

**Review by froch — APPROVED** [review]

**Pull request merged**